packages/filebrowser/src/listing.ts

```diff
@@ -1115,6 +1115,23 @@ export class DirListing extends Widget {
     if (!files || files.length === 0) {
       return;
     }
+    const length = event.dataTransfer?.items.length;
+    if (!length) {
+      return;
+    }
+    for (let i = 0; i < length; i++) {
+      let entry = event.dataTransfer?.items[i].webkitGetAsEntry();
+      if (entry.isDirectory) {
+        console.log('currently not supporting drag + drop for folders');
+        void showDialog({
+          title: this._trans.__('Error Uploading Folder'),
+          body: this._trans.__(
+            'Drag and Drop is not currently supported for folders'
```

Comment:
Not a native speaker but I feel

`Drag and Drop is currently not supported for folders` sounds better.

Reply:
thanks for the review! I updated the text, agree that the re-wording sounds a bit better!